R/pkg/R/generics.R

```diff
@@ -691,11 +691,11 @@ setGeneric("contains", function(x, ...) { standardGeneric("contains") })
 
 #' @rdname column
 #' @export
-setGeneric("desc", function(x) { standardGeneric("desc") })
+setGeneric("endsWith", function(x, suffix) { standardGeneric("endsWith") })
```

Comment:
can you move this back into the existing order ? it is sorted alphabetically if i'm not wrong.